opencog/learning/miner/MinerUtils.h

```diff
@@ -421,6 +421,24 @@ class MinerUtils
 	                                    const HandleSet& texts,
 	                                    unsigned ms);
 
+	/**
+	 * Return an atom to serve as key to store the support value.
+	 */
+	static const Handle& support_key();
```

Comment:
Good point, though I'm not sure it should be moved to a private part because I might add a scheme binding to it in the future, to grab pattern support, etc. Alternatively I could add a binding for `get_pattern`, etc, I'm just not sure which road I'm gonna take yet.